reference/commands/create.rst

```diff
@@ -59,6 +59,34 @@ The same happens for lockfiles created with ``--lockfile-out`` argument. The loc
   dependencies, you can control their build using the ``--build-test`` argument.
 
 
+Methods execution order
+-----------------------
+
+The ``conan create`` executes :ref:`methods <reference_conanfile_methods>` of a *conanfile.py* in the following order:
```

Comment:
When this is properly ordered, note that you can use the `graphviz` directive to generate graphs on the fly, see https://www.sphinx-doc.org/en/master/usage/extensions/graphviz.html